### PR TITLE
Adapt to bevy latest (#2)

### PR DESCRIPTION
**Pull request by HydrogenC**
Same as #2.

### CODE REVIEW

**Comment by HydrogenC** [comment]
It still doesn't pass the CI, I can't infer why. :(

**Comment by sburris0** [comment]
It looks like there were some syn problems upstream: https://github.com/bevyengine/bevy/commit/9193fc57d04fb492ac2b60cd75445930423f4425

If `cargo update` doesn't fix anything, I'll just wait until this gets sorted out to merge

**Comment by sburris0** [comment]
Might as well set the version to 0.5 now that it's out

File: src/lib.rs
Comment (BlackPhlox) [inline src/lib.rs:59]: I ran your changes locally and get a build error here:
```
error[E0599]: no method named `with` found for struct `EntityCommands<'_, '_>` in the current scope
  --> src\lib.rs:59:10
   |
59 |         .with::<FlyCam>();
   |          ^^^^ method not found in `EntityCommands<'_, '_>`
```
Should be?:
```
.insert(FlyCam);
```
(Source: https://bevyengine.org/news/bevy-0-5/#entityref-entitymut)

**Review by BlackPhlox — CHANGES_REQUESTED** [review]

**Comment by HydrogenC** [comment]
Closed as there is a more complete PR (#4), but weirdly, it still doesn't pass the CI.

**Pull request closed without merging**
